styles/globals.css

```diff
@@ -354,4 +354,34 @@ abbr[title] {
 
 select {
   cursor:pointer;
-}
\ No newline at end of file
+}
+.dark .gradient-border {
+  border: 2px solid transparent;
+  border-image: linear-gradient(225deg, #AD20E2 0%, #2DCCFD 100%);
+  border-image-slice: 1;
+  border-radius: 16px; /* <-- make it more rounded */
+}
+@layer base {
+  :root {
+    --background: 0 0% 100%;
```

Comment:
Please add color name comments to all these variables.

Reply:
done